test/parallel/test-fs-read-stream-patch-open.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const common = require('../common');
+const fs = require('fs');
+
+common.expectWarning(
+  'DeprecationWarning',
+  'ReadStream.prototype.open is deprecated', 'DEP0XXX');
+const s = fs.createReadStream('asd').on('error', () => {});
```

Comment:
Should the `on('error')` callback be a `common.mustCall()`

Reply:
This test doesn't care whether there is an error or not.

Reply:
Then I would recommend adding a comment saying that so that someone coming in later is not confused